packaging/packager_deb.sh

```diff
@@ -38,6 +38,7 @@ pip3 install vncdotool==0.12.0
 apt-get -y install xdotool x11-apps imagemagick
 apt-get -y install python3-tk scrot
 pip3 install pyautogui==0.9.53
+pip3 install pyscreeze==0.1.27
```

Comment:
Nice!

Reply:
The Linux distros was having version conflicts due to pyscreeze not being installed correctly thus I have to include an additional install command

Reply:
By "not installed correctly" you mean not installed at all or some wrong version? In any case this is a great fix, glad to have it.

Reply:
The original error was `pyautogui.PyAutoGUIException: PyAutoGUI was unable to import pyscreeze.` so the simple way to fix this is to include `pip3 install pyscreeze` into the code base on this [link](https://stackoverflow.com/questions/73925578/pyautogui-was-unable-to-import-pyscreeze)

Reply:
I see and so far what you say makes sense. What I wonder though is why it used to work before. A quick research shows that `scrot` is also used for taking screenshots and was installed right before PyAutoGUI meaning that it is likely the screenshot utility for PyAutoGUI we used before. Now for some reason it seems to have failed. Could it be that PyAutoGUI changed their dependencies? Could we try to remove the `scrot` dependency entirely to just have `pyscreeze` now and do the tests pass then with just the latter?

Reply:
I see you pushed some commits to revert the older pyautogui version? Did you have any findings from these changes?

Reply:
I notice there has been many threads regarding the changes in dependency between pyautogui version 0.9.5x and earlier versions such as 0.9.41. So I might try to revert it back to 0.9.41 to see if it fixes the  `pyscreeze` problem, but it seems that our repository is based on the pyautogui 0.9.53 thus it return dependency error: `ModuleNotFoundError: No module named 'Xlib'`

Reply:
So `scrot` and `pyscreeze` are not alternatives? Maybe PyAutoGUI's documentation and release versions explain this somewhere on the internet? It seems to me that both of them are used for screenshots, maybe we previously used scrot and now we can just use your newly suggested pyscreeze?

Reply:
Resolving this one since we now know the PyScreeze is incompatible with the most current Pillow version on Fedora/Ubuntu containers and has to be forced to a lower one.